docs-source/content/userguide/managing-domains/model-in-image/model-files.md

```diff
@@ -0,0 +1,156 @@
+  ServerStartMode: 'prod'
+
+topology:
+  Name: '@@ENV:DOMAIN_UID@@'
+  AdminServerName: "admin-server"
+  Cluster:
+    "cluster-1":
+      DynamicServers:
+        ServerTemplate:  "cluster-1-template"
+        ServerNamePrefix: "managed-server"
+        DynamicClusterSize: 5
+        MaxDynamicClusterSize: 5
+        CalculatedListenPorts: false
+  Server:
+    "admin-server":
+      ListenPort: 7001
+  ServerTemplate:
+    "cluster-1-template":
+      Cluster: "cluster-1"
+      ListenPort: 8001
+```
+
+Some notes about the sample model file:
+ - It includes a WebLogic credentials stanza that is required by Model in Image.
+ - It derives its domain name from the pre-defined environment variable `DOMAIN_UID` but note that this is not required.
+ - For a description of model file macro references to secrets and environment variables, see [Model file macros](#model-file-macros).
+
+#### Important notes about Model in Image model files
+
+- Understand when to use model macros.
+
+  - You can use model macros to reference arbitrary secrets from model files. This is recommended for handling mutable values such as database user names, passwords, and URLs. See **[Using secrets in model files](#using-secrets-in-model-files)**.
+
+    - All password fields in a model should use a secret macro. Passwords should not be directly included in property or model files because the files may appear in logs or debugging. 
+
+    - Model files encrypted with the [WDT Encrypt Model Tool](https://github.com/oracle/weblogic-deploy-tooling/blob/master/site/encrypt.md) are not supported. Use secrets instead.
+
+  - You can use model macros to reference arbitrary environment variables from model files. This is useful for handling plain text mutable values that you can define using an `env` stanza in your domain resource, and is also useful for accessing the built in `DOMAIN_UID` environment variable. See [Using environment variables in model files](#using-environment-variables-in-model-files).
+
+  - For most models, it's useful to minimize or eliminate the usage of model variable files (also known as property files) and use secrets or environment variables instead.
+
+- A model __must__ contain a `domainInfo` stanza that references your WebLogic administrative credentials. You can use the `@@SECRET` macro with the reserved secret name `__weblogic-credentials__` to reference your domain resource's WebLogic credentials secret for this purpose. For example:
+
+    ```
+    domainInfo:
+      AdminUserName: '@@SECRET:__weblogic-credentials__:username@@'
+      AdminPassword: '@@SECRET:__weblogic-credentials__:password@@'
+    ```
+- You can control the order that WDT uses to load your model files, see [Model file naming and loading order](#model-file-naming-and-loading-order).
+
+#### Model file naming and loading order
+
+Refer to this section if you need to control the order in which your model files are loaded.
```

Comment:
how do i know if i need that?

Reply:
@markxnelson Please review https://github.com/oracle/weblogic-kubernetes-operator/pull/1510   I will resolve once the change is merged.